sorc/build_ufs.sh

```diff
@@ -5,16 +5,17 @@ cwd=$(pwd)
 
 # Default settings
 APP="S2SWA"
-CCPP_SUITES="FV3_GFS_v17_p8_ugwpv1,FV3_GFS_v17_coupled_p8_ugwpv1"  # TODO: does the g-w need to build with all these CCPP_SUITES?
+CCPP_SUITES="FV3_GFS_v17_p8_ugwpv1,FV3_GFS_v17_coupled_p8_ugwpv1,FV3_global_nest_v1"  # TODO: does the g-w need to build with all these CCPP_SUITES?
 PDLIB="ON"
 
-while getopts ":da:j:vw" option; do
+while getopts ":da:j:vwf" option; do
```

Comment:
These should be in alphabetical order as well. I recognize the `d` was already out-of-order, so just put the `f` after the `a:` and I'll fix the rest of the order later.

Reply:
sounds good. thanks!